frb_example/pure_dart/rust/src/api.rs

```diff
@@ -531,6 +532,17 @@ pub fn get_message() -> ApplicationMessage {
     external_lib::poll_messages()[1].clone()
 }
 
+#[frb(mirror(Numbers, Sequences))]
+pub struct _Numbers(pub Vec<i32>);
+
+pub fn repeat_number(num: i32, times: usize) -> Numbers {
+    external_lib::repeat_number(num, times)
+}
+
+pub fn repeat_sequence(seq: i32, times: usize) -> Sequences {
+    external_lib::repeat_sequences(seq, times)
+}
+
```

Comment:
Would be great if there are a little bit more tests. For example: Functions that *accept* Numbers/Sequences as arguments. This is because rust functions that *accept arguments of type T* is implemented very differently from *return type T*.